app/src/main/java/com/moviefiend/torian/moviefiend/MovieAdapter.java

```diff
@@ -0,0 +1,55 @@
+package com.moviefiend.torian.moviefiend;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.moviefiend.torian.moviefiend.network.NowPlayingResponse;
+
+import java.util.ArrayList;
+
+
+public class MovieAdapter extends RecyclerView.Adapter<MovieViewHolder> {
+
+    private Context mContext;
+    private ArrayList<NowPlayingResponse.MovieInfo> mMovies;
```

Comment:
You should always use the interface/base class for the reference type (e.g. `List` and not `ArrayList`).